exonum-java-light-client/README.md

```diff
@@ -0,0 +1,30 @@
+# Exonum Java Light Client
+Java client for [Exonum blockchain][exonum].
```

Comment:
Shall we put somewhere here an overview of its intended functionality (what exactly the light client is and what it allows to do)?

Reply:
yes, sure. But I'm going to do that after P1 development, to write the readme once along with getting started guide, examples etc.

Reply:
Makes sense!